activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Fix a regression with division by a Duration object
+
+    Before:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 2700.0 hours
+    
+    After:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 0.75
```

Comment:
To illustrate:

```
>> today = DateTime.current
=> Sat, 15 Jul 2017 05:27:47 +0000
>> tomorrow = today + 1.day
=> Sun, 16 Jul 2017 05:27:47 +0000
>> ((tomorrow - today) / 1.hour).to_i
=> 3600
>> ((tomorrow - today).days / 1.hour).to_i
=> 24
```

There is still a bug though with how the division is assuming seconds, e.g:

```
>> today = Time.current
=> Sat, 15 Jul 2017 05:33:47 UTC +00:00
>> tomorrow = today + 1.day
=> Sun, 16 Jul 2017 05:33:47 UTC +00:00
>> ((tomorrow - today) / 1.hour).to_i
=> 311040000
>> ((tomorrow - today).seconds / 1.hour).to_i
=> 24
```

We can safely return dimensionless numbers where both are instances of ActiveSupport::Duration - it's probably okay for instances where the numerator is a plain numeric too as I can't think of a scenario where it wouldn't be.